### PR TITLE
add spatext as dependency

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
This PR reminds me that I have a few bugs in handling math tags, will follow up with that later.

**Comment by brandon-sisler** [comment]
I'll pull the trigger on literally anything you send me, but what is a dependency in this case?

**Comment by StevenClontz** [comment]
I'll post a code review after dinner to explain

**Comment by brandon-sisler** [comment]
Good enough for me, time to merge

File: README.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ npm create svelte@latest my-app
 Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
 
 ```bash
-npm run dev
+npm run dev -- --host
```
Comment (StevenClontz) [inline README.md:22]: I just moved the `--host` option here...

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
 	"version": "0.0.1",
 	"private": true,
 	"scripts": {
-		"dev": "vite dev --host",
```
Comment (StevenClontz) [inline package.json:6]: ... from here.

File: package.json [hunk 3]
```diff
@@ -33,5 +33,8 @@
 		"vite": "^4.0.0",
 		"vitest": "^0.25.3"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"spatext": "^0.3.0-beta.20230312"
```
Comment (StevenClontz) [inline package.json:38]: I installed the `spatext` Node package that I just published to https://www.npmjs.com/package/spatext

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -10,12 +11,17 @@
...
 </script>
 
 <!-- <input type='number' bind:value={height}/> -->
 <BlocklyComponent {height} width={1200} {toolbox} bind:workspace/>
 
 {#if workspace}
-<textarea id="blocklyProof" style="height: 320px; width: 600px;" readonly>{SpatextGenerator.workspaceToCode(workspace)}</textarea>
+<textarea id="blocklyProof" style="height: 320px; width: 600px;" readonly>{stx}</textarea>
 <textarea id="blocklyCoq" style="height: 320px; width: 600px;" readonly>{CoqGenerator.workspaceToCode(workspace)}</textarea>
+<SpaTeXt {stx}/>
```
Comment (StevenClontz) [inline src/routes/+page.svelte:26]: Now you have access to a `SpaTeXt` component that takes the string `stx` and spits out pretty (imho) typeset mathematicds.

**Pull request merged**